code/src/ThoughtContainer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import moment from 'moment';
+
+
+export const ThoughtContainer = ( {message, created, hearts, id, onThoughtLiked} ) => {
```

Comment:
Great that you're using destructuring of the props (which I didn't), makes it more readable